### ISSUE TITLE
Climate turn_off not supported

### ISSUE THREAD

**Issue by MelleD** [tschamm/boschshc-hass#196]
I get follow error with climate gen 2.

"Die Aktion climate/turn_off konnte nicht ausgeführt werden. Entity climate.room_climate_xxx does not support this service."

I'm sure it worked once. How else can I turn it off?

**Comment by cars1806** [comment]
Hi, use "climate.set_hvac_mode" and turn it on and off through commands like "heat" or "off" (depends on your hvac system).
I also had to change my automations because turning it on and off didn't work anymore, but this seems to be the trick.

**Comment by MelleD** [comment]
Yes meanwhile I wrote own scripts/actions to toogle the state etc.

```
heat_toggle:
  sequence:
    - service: climate.set_hvac_mode
      data:
        hvac_mode: >
          {% if states('climate.room_climate_xxx') == "off" %}
          heat
          {% else %}
          off
          {% endif %}
      target:
        entity_id: climate.room_climate_xxx
  mode: single
```

But direct support is also valuable. Thats why I let this fr open :)

**Comment by edzards** [comment]
Would be great if this works as before. With a voice assistant it was very convenient to use the turn_on and turn_off services.

**Comment by tschamm** [comment]
Fixed in #211, thanks to @dingausmwald

**Comment by tschamm** [comment]
Set temperature seems to be non functional

**Comment by dingausmwald** [comment]
Yea, don't know how this could happen, pushed a fix this morning already. Anyway. set_hvac_mode changed to async_set_hvac_mode see #224

**Comment by mosandlt** [comment]
This is fixed in the current releases (0.4.104+). The climate entity now declares `ClimateEntityFeature.TURN_OFF | TURN_ON` and exposes `HVACMode.OFF`, so `climate.turn_off` (and `turn_on`) work directly, in addition to the `climate.set_hvac_mode` → `off`/`heat` path.

Please update via HACS to 0.4.106 and restart HA. Reopen if `climate.turn_off` still reports "not supported" for your thermostat.

**Comment by mosandlt** [comment]
Fixed in **boschshc-hass 0.4.104** — climate.turn_off / HVACMode.OFF is now supported.

Please update (via HACS, or manually — see #279) and restart Home Assistant. If you're on ≥0.4.104 and still hit this, comment here and we'll dig back in; otherwise I'll close this in ~2 weeks. Thanks!

**Comment by mosandlt** [comment]
`climate.turn_off` / `turn_on` are supported — the climate entity advertises `TURN_OFF`/`TURN_ON` and implements `async_turn_on/off`.

Available in current releases — please update to **boschshc-hass 0.7.6**. Closing; reopen if anything is missing.
